modules/ipi-install-troubleshooting-bootstrap-vm-inspecting-logs.adoc

```diff
@@ -14,10 +14,7 @@ bootstrapOSImage: http://<ip:port>/rhcos-43.81.202001142154.0-qemu.<architecture
 clusterOSImage: http://<ip:port>/rhcos-43.81.202001142154.0-openstack.<architecture>.qcow2.gz?sha256=a1bda656fa0892f7b936fdc6b6a6086bddaed5dafacedcd7a1e811abb78fe3b0
 ----
 
-The `ipa-downloader` and `coreos-downloader` containers download resources from a webserver or the external link:https://quay.io[quay.io] registry, whichever the `install-config.yaml` configuration file specifies. Verify the following two containers are up and running and inspect their logs as needed:
-
-* `ipa-downloader`
-* `coreos-downloader`
+The `coreos-downloader` container downloads resources from a webserver or the external link:https://quay.io[quay.io] registry, whichever the `install-config.yaml` configuration file specifies. Verify that the `coreos-downloader` container is up and running and inspect its logs as needed.
```

Comment:
We could even be more specific: it downloads CoreOS boot artifacts (ISO and kernel/initramfs for PXE).

Reply:
@dtantsur PTAL at revised content [here](https://github.com/openshift/openshift-docs/pull/57028/files#diff-ea2985fb173a52686f6612b16fddecbe56fe1b3fdf2424408fa96356249cef22R17). Thank you!